crates/sui-core/src/authority/authority_store_tables.rs

```diff
@@ -93,3 +94,17 @@ pub struct AuthorityStoreTables<S> {
     /// every message output by consensus (and in the right order).
     pub(crate) last_consensus_index: DBMap<u64, ExecutionIndices>,
 }
+
+// These functions are used to initialize the DB tables
+fn objects_table_default_config() -> Options {
```

Comment:
Wait, how are these different from the default options?  Can we just use one function for all of them?  No need to repeat just for having a diff name for each table

Reply:
@velvia yes we can indeed use one function. I made them explicit just to be defensive so that whoever is modifying the code doesn't accidentally modify another table's configs. But yes in a perfect world, it's all same.